docs/Getting Started.md

```diff
@@ -1,3 +1,70 @@
+## Setup Guide
+
+### Preliminary decisions
+Before you begin setting up your new project, you'll need to make a few important decisions about how you use this project and set some repo variables accordingly.
+
+#### Branch naming conventions
+The workflows in this repo execute only on branches whose names match certain user-configurable patterns.  For example, when issue branches are created, dev and UAT sandboxes will be created.  And when pull requests are opened against a release branch, the pull request will be deployed to a UAT org.  Therefore, you'll need to decide upon the branch name prefixes that those workflows will use to identify issue and release branches.  Once you've decided on your naming conventions, you'll need to set repository variables to store the branch prefixes.  If you're unsure what values to use, we recommend the following:
+| Branch type | Prefix | Repo variable name |
+| ----------- | ------ | ------------------ |
+| Issue       | issue- | ISSUE_BRANCH_PREFIX |
+| Release     | REL-   | RELEASE_BRANCH_PREFIX |
+
+#### Profile and permissionset format
+If enabled in this project, an [sfdx plugin](https://www.npmjs.com/package/@rdietrick/sfdx-profile-decompose) can translate monolithic Salesforce profiles and permissionsets into more granular XML files in your project.  We've found that this makes managing diffs and conflict resolution in these files much easier.  When the plugin is enabled, each profile will, for instance, be broken down into separate files for each object that the profile maintains FLS for.  To turn it on, set the repository variable `SALESFORCE_FORMATTED_PROFILES_AND_PERMS` to true.  If the plugin is disabled, you will also need to remove or comment out the line in `scripts/retrieve` that executes the profiles:decompose plugin.  If you choose to store only profiles or only permissionsets in your repo but wish to use this plugin, you'll need to pass the `--md-types=profiles` or `--md-types=permissionsets` argument to the profiles:decompose command in the retrieve script.
+
+#### Release tracking
+If enabled, each time a release branch is merged to your main branch, a [release](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository) will be automatically created in your repo with release notes for each issue included in the release.  To turn this on, set the repository variable `GENERATE_RELEASE` to "true."
```

Comment:
We may want to use `true` instead of “true.” so people do not think they need to add a period in the Actions variable value